jboss-kie-kieserver/added/launch/jboss-kie-kieserver-jms.sh

```diff
@@ -115,4 +117,15 @@ function configureJmsAudit() {
             echo -e "\njbpm.audit.jms.transacted=false" >> "${KIE_AUDIT_PROPERTIES_FILE}"
         fi
     fi
+}
+
+function configureJmsExecutorMdb(){
+    if [ -n "${JBOSS_MDB_MAX_SESSIONS}" ];then
+        log_info "Configuring KieServerExecutorMDB Max Sessions on ejb-jar.xml"
+        sed -i  's#<activation-config-property-value>queue/KIE.SERVER.EXECUTOR</activation-config-property-value>#<activation-config-property-value>queue/KIE.SERVER.EXECUTOR</activation-config-property-value>\
```

Comment:
the only thing missing from previous reviews is to improve this `sed`;
try to use a file marker like ' <!-- ##EXECUTOR_MAX_SESSION## -->'

Reply:
Fixed, now the output of cat is 
```xml
        <message-driven>
            <!-- we need to filter the JMS properties to override what is in KieExecutorMDB -->
            <ejb-name>KieExecutorMDB</ejb-name>
            <ejb-class>org.kie.server.jms.executor.KieExecutorMDB</ejb-class>
            <transaction-type>Bean</transaction-type>
            <activation-config>
                <activation-config-property>
                    <activation-config-property-name>destination</activation-config-property-name>
                    <activation-config-property-value>queue/KIE.SERVER.EXECUTOR</activation-config-property-value>
                </activation-config-property>
                <activation-config-property>
                    <activation-config-property-name>maxSession</activation-config-property-name>
                    <activation-config-property-value>1234567</activation-config-property-value>
                </activation-config-property>
                <activation-config-property>
                    <activation-config-property-name>destinationType</activation-config-property-name>
                    <activation-config-property-value>javax.jms.Queue</activation-config-property-value>
                </activation-config-property>
                <activation-config-property>
                    <activation-config-property-name>acknowledgeMode</activation-config-property-name>
                    <activation-config-property-value>Auto-acknowledge</activation-config-property-value>
                </activation-config-property>
            </activation-config>
        </message-driven>
```

Reply:
I think my last comment was missing the last part.

about the _sed_ command, please try to add a file marker on the ejb-jar.xml instead _sed_ its content.

add something like <! ##MAX_SESSSION## -->
and sed this file marker instead the file content.

it is unlikely to happen new changes on this file, but if that does happen, we don't need to update the sed command later.